src/brevitas_examples/imagenet_classification/ptq/utils.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import torchvision
+
+from brevitas import config
+from brevitas_examples.imagenet_classification import models
+
+config.IGNORE_MISSING_KEYS = True
+
+MODEL_LIST = {
```

Comment:
this isn't a list and it doesn't tell me what information the dictionary reports, call it MODEL_TOP1_IMAGENET_MAP